lib/rubocop/cop/style/hash_except.rb

```diff
@@ -13,6 +13,9 @@ module Style
       # when used `==`.
       # And do not check `Hash#delete_if` and `Hash#keep_if` to change receiver object.
       #
+      # @safety
+      #   This cop is unsafe because the receiver may not be a Hash.
```

Comment:
Can you tweak it?

```suggestion
      #   This cop is unsafe because it cannot be guaranteed that the receiver
      #   is a `Hash` or responds to the replacement method.
```

Reply:
Thanks, that's much better 👍